exodus_lambda/functions/origin_response/origin_response.py

```diff
@@ -60,4 +61,4 @@ def handler(self, event, context):
 
 
 # Make handler available at module level
-lambda_handler = LambdaClient().handler
+lambda_handler = LambdaClient().handler  # pylint: disable=invalid-name
```

Comment:
I know this isn't the first time this has been added... but I don't really understand why this is needed.  pylint shouldn't have a problem determining that an instance of LambdaClient has a 'handler' property.  pylint as run by Travis also doesn't seem to need this.

It's not particularly important, but you might want to consider looking at the cause of this in your environment as it could be indicating some setup problem.

Reply:
`LambdaClient().handler` isn't the issue
> exodus_lambda/functions/origin_response/origin_response.py:69:0: C0103: Constant name "lambda_handler" doesn't conform to UPPER_CASE naming style (invalid-name)

Not sure what specifically about my environment could be causing the complaint, as it's just the dev-env Python3 virtualenv.